dev/bots/test.dart

```diff
@@ -273,6 +273,11 @@ Future<void> _runToolTests() async {
   await selectSubshard(subshards);
 }
 
+// Example apps that should not be built by _runBuildTests`
+const List<String> _excludedExampleApplications = <String>[
+  'splash',
```

Comment:
Could you test it without --fast-start?

Could you add a comment explaining the reason to skip it?

Reply:
Added a comment explaining the skip: there is nothing to build for this app